src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,56 @@
+var timeBound = ['', '1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
+var currencies = ['usd']
+
+function optionsChecker (options) {
+  if ('currency' in options) {
+    options.currency = options.currency || 'usd'
+    if (currencies.indexOf(options.currency) === -1) {
```

Comment:
The error will never be thrown because you are setting a default value by:
```js
options.currency = options.currency || 'usd'
```
Split the function into `currencyOptionChecker_` & `timeboundOptionChecker_` and call them from `optionsChecker`.
Add tests for these functions, they are pretty important. 
Also they should be "private".

Reply:
I did as you said